pkgs/by-name/hi/hickory-util/package.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  rustPlatform,
+  fetchCrate,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "hickory-util";
+  version = "0.24.1";
+
+  src = fetchCrate {
+    inherit pname version;
```

```suggestion
    pname = "hickory-util";
    inherit version;
```
I think `src` `pname`s or `repos` should be static.